catalyst_voices/packages/internal/catalyst_voices_blocs/lib/src/campaign/info/campaign_info_cubit.dart

```diff
@@ -0,0 +1,28 @@
+import 'dart:async';
+
+import 'package:catalyst_voices_blocs/src/campaign/info/campaign_info.dart';
+import 'package:catalyst_voices_services/catalyst_voices_services.dart';
+import 'package:catalyst_voices_shared/catalyst_voices_shared.dart';
+import 'package:catalyst_voices_view_models/catalyst_voices_view_models.dart';
+import 'package:flutter_bloc/flutter_bloc.dart';
+
+/// Gets the campaign info.
+final class CampaignInfoCubit extends Cubit<CampaignInfoState> {
+  final CampaignService campaignService;
+
+  CampaignInfoCubit({required this.campaignService})
+      : super(const CampaignInfoState(isLoading: true));
```

Comment:
umm this is wrong. You should set it only inside `load` to **true** at the beginning and **false** at the end.

Reply:
probably `copyWith` will be needed

Reply:
I changed it to set `isLoading=true` whenever `load` is called

Reply:
the default constructor param of the state for `isLoading` is false so it's done already (setting isLoading to false at the end of `load`)